### ISSUE TITLE
Add JSON Schema

### ISSUE THREAD

**Issue by nichtich** [LinkedPasts/linked-places#8]
The structure of this format seems to be documented only through examples and the documentation (e.g. which fields are required, what they must contain...). An additional JSON Schema may help to validate records and guide data creation and curation.

**Comment by rsimon** [comment]
Hi @nichtich, I think all of this will come, with time :-) (@kgeographer will know more.) But for the mean time, there is this (interim?) schema that some have started to use for validation:

https://github.com/kgeographer/whgazetteer/blob/master/datasets/static/lpf-schema.json

**Comment by kgeographer** [comment]
Yes, the link is to the last schema I used internally for WHG. The LP model has changed slightly since, so I need to a) update for my internal use, and b) create a CLI python script so anyone can do it. It's a little tricky because so many elements are optional; I'm on the move and expect to have this done within a few weeks. The LP model has seen slight modifications in response to more examples of contributor data.
